OpenRA.Mods.Common/Scripting/Global/MediaGlobal.cs

```diff
@@ -168,24 +169,24 @@ public bool PlayMovieInRadar(string movie, LuaFunction playComplete = null)
 		}
 
 		[Desc("Display a text message to the player.")]
-		public void DisplayMessage(string text, string prefix = "Mission")
+		public void DisplayMessage(string text, string prefix = "Mission", bool useCustomColor = false, HSLColor color = new HSLColor())
```

Comment:
I agree with WolfGaming here. Set the default to Color.White and remove `useCustomColor`.